pylzy/tests/env/explorer/test_classify.py

```diff
@@ -58,9 +59,37 @@ def test_classify_pypi_packages(classifier: ModuleClassifier, pypi_index_url: st
             name='sampleproject',
             version='3.0.0',
             pypi_index_url=pypi_index_url,
+            platform_present=True,
         )
     })
 
+    @dataclasses.dataclass
+    class MyDistribution:
+        name: str
+        version: str
+
+    assert classifier._classify_distributions([  # type: ignore
```

Comment:
is it use internet? i thought we try to avoid it in unit tests
although it is good to see concrete working examples here